packages/babel-preset-expo/index.js

```diff
@@ -1,16 +1,42 @@
+const lazyImportsBlacklist = require('./lazy-imports-blacklist');
+
 module.exports = function(api, options) {
   const isWeb = api.caller(isTargetWeb);
 
   if (isWeb) {
     return getWebConfig(options.web);
   }
 
-  return getNativeConfig();
+  return getNativeConfig(options);
 };
 
-function getNativeConfig() {
+function getNativeConfig(options) {
+  // Note that if `options.lazy` is not set (i.e., `null` or `undefined`),
+  // `metro-react-native-babel-preset` will handle it.
+  const lazyOption = options && options.lazy;
   return {
-    presets: ['module:metro-react-native-babel-preset'],
+    presets: [
+      [
+        // TODO: use `module:metro-react-native-babel-preset` after this package is updated to at least v0.53.1 in the root `node_modules`.
+        // Ref: https://github.com/expo/expo/pull/4685#discussion_r296462573
+        require('metro-react-native-babel-preset'),
```

Comment:
Maybe let's always do this, even if we upgrade the preset elsewhere. It's not good if the presets can change unintentionally.